nntrainer/models/neuralnet.cpp

```diff
@@ -857,12 +857,14 @@ std::vector<float *> NeuralNetwork::incremental_inference(
               last_out_buf_data + batch * out_t.width(), 1);
 
 #else
+        throw std::invalid_argument("Errro: enable-fp16 is not set");
+#endif
+      } else {
```

Comment:
maybe it would be better to specify if the data type is fp32 
```suggestion
      } else if (out->getDataType() == ml::train::TensorDim::DataType::FP32) {
```

Reply:
Done